README.md

```diff
@@ -68,14 +68,22 @@ An object with the same interface as `THREE.Mesh` but used to evaluate CSG opera
 > 
 > It is recommended to remove groups from a geometry before creating a brush if multi-material support is not required.
 
-<!--
 ## Operation
 
 _extends Brush_
 
-TODO
+This is an extension of the `Brush` class. With this class is possible to create a mesh and pass the CSG Operation constant to define which operation computes on the mesh. If you don't choose an operation, the default is `ADDITION`. The result is a Mesh whose effect is defined from the operation selected. You can see how it works in the _hierarchy_ example.
 
--->
+```js
+const root = new Operation( new THREE.BoxBufferGeometry(10, 5, 0.5), new GridMaterial() );
+```
```

Comment:
I don't think this is needed - the implicit constructor of `Brush` should be enough, I think!